projects/plugins/social/src/js/components/toggle-section/styles.module.scss

```diff
@@ -59,8 +59,10 @@
 	margin-top: calc( var( --spacing-base ) * 2 );
 	grid-column: span 2;
 	justify-self: flex-start;
+	width: 100%;
```

Comment:
The `<Button>` component now has [a `fullWidth` property](https://automattic.github.io/jetpack-storybook/?path=/story/js-packages-components-button--full-width). I think using this in combination with [the `isBreakpointMatch` hook](https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/components/components/layout/use-breakpoint-match) is the preferred way to add viewport-dependent styling.

You could do something like this:

```js
import { useBreakpointMatch } from '@automattic/jetpack-components';
const [ isLg ] = useBreakpointMatch('lg')

<Button fullWidth={ ! isLg }>
```

I'm not sure if `lg` is the proper breakpoint here, there's also `md` and `sm`.

_(I didn't know this when I initially made the admin page, which is why some components use media queries instead. We should probably do this for all media queries, but that's for another PR 😄)_

Reply:
Thanks for the tip, I didn't know about this! I changed it, looks like `sm` is the one that applies for smaller than 600px, so that worked as intended! 👍